staging/src/k8s.io/apiserver/pkg/server/options/authentication.go

```diff
@@ -36,6 +37,17 @@ import (
 	openapicommon "k8s.io/kube-openapi/pkg/common"
 )
 
+// DefaultAuthWebhookRetryBackoff is the default backoff parameters for
+// both authentication and authorization webhook used by the apiserver.
```

Comment:
If a webhook returns a 5xx response that has `Retry-After` set, the client end (Kubernetes) does not pay attention to that.

Would it be OK to briefly mention in a comment that this implementation does not adjust behavior based on `Retry-After`?
(to be clear - it's OK not to check that header; maybe a future PR can start checking that response header)